srcjs/updateGraph/searchString.js

```diff
@@ -58,7 +58,11 @@ let searchStringWithData = function(obj: Node | Edge | GhostEdge) {
 let searchStringContainer = function(searchElement_: JQuery) {
   searchElement = searchElement_;
   searchElement.on("input", function(e) {
-    searchStringWith($(e.target).val());
+    searchStringWith(
+      $(e.target)
+        .val()
+        .toString()
```

Comment:
Why's this? Would `val()` ever return not a string?

Reply:
This was a flow issue.  I agree it's always a string, but flow wants to make sure of it.